tests/manage/z_cluster/nodes/test_toleration.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import pytest
+
+import re
+
+from ocs_ci.framework.testlib import (
+    tier4,
+    tier4a,
+    E2ETest,
+)
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.machine import get_labeled_nodes
+from ocs_ci.ocs.resources.pod import (
+    get_all_pods,
+    delete_pods,
+    validate_pods_are_respinned_and_running_state,
+)
+from ocs_ci.ocs.ocp import OCP
+
+
+logger = logging.getLogger(__name__)
+
+
+def check_toleration_on_pods(toleration_key=None):
```

Comment:
As it's related to pod, we might to move this to:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pod.py